src/deploy/osp_deployer/deployer.py

```diff
@@ -233,6 +233,7 @@ def validate_edge_sites_in_settings(args, setts):
 
 
 def deploy_powerflex_gw(setts, sah_node, tester, powerflexgw_vm):
+    settings = setts
```

Comment:
Is this line needed? we're passing settings as setts, and is called as such afterward.

Reply:
see a couple of lines further down : 
settings.powerflexgw_vm.root_password
is called a few times

Reply:
should be setts.powerflexgw_vm.root_password. I figure I missed to change it.

Reply:
done